src/NUnitFramework/framework/Internal/Execution/SimpleWorkItem.cs

```diff
@@ -153,6 +154,10 @@ private TestCommand MakeTestCommand()
                     command = new TimeoutCommand(command, timeout);
 #endif
 
+                // Add wrappers for repetable tests after timeout so the timeout is reset on each instance
+                foreach (var repeatableAttribute in method.GetAttributes<IRepeatTest>(true))
+                    command = repeatableAttribute.Wrap(command);
```

Comment:
What happens when an attribute implements both `IWrapSetUpTearDown` and `IRepeatTest`? I think it'll get applied both here and in the earlier loop. We should add a test on this behavior.